File: rust/day6.rs
fn main() {
    let str1: String = "Hello".to_string(); // or use let str1 = "Hello";
    let int1: u32 = 11232532;
    let float1: f32 = 11232312.99999;
    let bool1: bool = false;

    println!(
        "{}",
        str1
        + "-" +
        &int1.to_string()
        + "-" +
        &float1.to_string()
        + "-" +
        &bool1.to_string()
    );
}

File: cSharp/day6.cs

using System;
using System.Globalization;

string str1 = "juan DELA cruz";
int int1 = 1111999;
double float1 = 764513.999;
bool bool1 = false;


System.Console.WriteLine(
    str1 + "-" + 
    int1.ToString()
    + "-" 
    + string.Format("{0:N3}",float1)  
    + "-" + bool1
);

